manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
I'd prefer to do it in another PR, I'll make a ticket for it. To Russ's comment, what we're doing with the class in question seems to be trying to give users the least barrier to entry in terms of getting a PuppetDB up and running, most intermediate/advanced users won't even be interacting with the class at all and will probably be managing postgres via it's own module. I think asking novice users to go look through documentation to find that they need to turn manage_repos on rather than telling advanced users they need to turn it off is probably not where we want to be.